platform/ios/Makefile

```diff
@@ -323,8 +330,8 @@ endif
 
 .PHONY: style-code
 style-code:
-	node scripts/generate-style-code.js
-	node scripts/generate-shaders.js
+	node ../../scripts/generate-style-code.js
+	# node ../../scripts/generate-shaders.js
```

Comment:
how come its disabled?

Reply:
I am getting an error when I run it, could you try it out?

Reply:
Removing the generated shaders is also out of scope for this PR, but I will get to that next.

Reply:
Great